templates/http-rust/content/src/lib.rs

```diff
@@ -1,12 +1,12 @@
-use spin_sdk::http::{IntoResponse, Request};
+use spin_sdk::http::{IntoResponse, Request, HeaderValue};
 use spin_sdk::http_component;
 
 /// A simple Spin HTTP component.
 #[http_component]
 fn handle_{{project-name | snake_case}}(req: Request) -> anyhow::Result<impl IntoResponse> {
-    println!("{:?}", req.headers);
+    println!("{:?}", req.headers().collect::<Vec<(&str, &HeaderValue)>>());
```

Comment:
Oh man... I don't want this to be the first thing a new user sees when they pop open a newly created Spin app...

Reply:
I guess we could turn it into a loop, which would also make for more readable output. But then the output is pretty verbose.  Maybe we should figure out what we're trying to achieve with this line.

Reply:
What if we printed one of the headers from https://developer.fermyon.com/spin/http-trigger#additional-request-information?  That would:

* Limit the amount of output
* Show how to access a specific header
* Teach people about the existence of the SPECIAL headers

E.g. `println!("Handling request to {}", req.header("spin-full-url"))` or something like that?

Reply:
Updated @itowlson